### DISCUSSION TITLE
Setting up SSL

### DISCUSSION THREAD

**Discussion by ftroot** [karakeep-app/karakeep#408] [Q&A] [answered]
Hi there,
Hope you're all well !
Not sure how and where to install my SSL certificate ? Could someone tell me please ?
Thanks !

**Comment by MohamedBassem** [comment, marked as answer]
hey, for ssl, you'll want to but hoarder behind a reverse proxy that takes care of the ssl for you (such as nginx proxy manager, caddy, etc). Hoarder itself doesn't deal with ssl certs.

**Reply by ftroot** [reply]
Thank you, Mohamed !
Will give Caddy a try as it seems to be an amazing tool.
Nice day and keep up the good job !
